### PR TITLE
Improved UX and UI

### PR DESCRIPTION
**Pull request by rares45**
I improved the aspect and feel.
Changes I made:
1. Added toast for visual feedback when needed
2. Changed a lot of icons
3. Rounded corners and changed colours
4. I moved all hardcoded strings to resources
5. Fixed bug (see commits)
6. Romanian translation

imageimage
imageimage

### CODE REVIEW

**Comment by rares45** [comment]
@slowscript what do you think about my changes?

**Review by slowscript — COMMENTED** [review]
Thanks for the PR. I like the changes overall, it's definitely a major improvement.
Here are a few nitpicks however:
- I think the "transfer" cards are a little too tall. Most of the design is pretty compact and this seems a bit out of place.
- There seems to be an issue with the icons. When I run the application in an emulator, they look fine, but in Android Studio and the PNGs they look irregular for some reason.
- I will also need you to *rebase* your branch onto my branch. You have instead *merged* my branch into yours, which means that if I merge this as-is, the commits I made will be duplicated.
- The remaining minor issues are in the comments

File: app/src/main/res/layout/remote_view.xml [hunk 1]
```diff
@@ -25,16 +25,18 @@
...
             app:layout_constraintTop_toTopOf="@+id/imgProfile"
-            app:srcCompat="@android:drawable/checkbox_on_background" />
+            app:srcCompat="@drawable/ic_status_connected" />
 
         <ImageView
             android:id="@+id/imgProfile"
             android:layout_width="48dp"
             android:layout_height="48dp"
             android:layout_marginStart="8dp"
+            android:tint="@color/iconTint"
```
[the remark this answers is not in the document]
Comment (rares45) [reply app/src/main/res/layout/remote_view.xml:35]: I don't know how I missed that. Gonna fix it

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,10 +1,51 @@
...
+    <string name="picture_settings_title">Profile picture</string>
+    <string name="download_dir_settings_title">Downloads directory</string>
+    <string name="notification_settings_title">Show notification about incoming files</string>
+    <string name="background_settings_title">Run in background</string>
+    <string name="group_code_settings_title">Group code</string>
+    <string name="port_settings_title">Port</string>
+    <string name="theme_settings_title">Theme</string>
+    <string name="overwriting_setting_title">Allow overwriting</string>
+    <!--  Theme settings -->
+    <string-array name="theme_settings_entries_array"> <!-- do not translate this -->
```
[the remark this answers is not in the document]
Comment (rares45) [reply app/src/main/res/values/strings.xml:32]: I did it this way initially. But it's only one array and it's a string one.

**Comment by rares45** [comment]
> ...When I run the application in an emulator, they look fine, but in Android Studio...

I have no idea what causes that. I tried fixing it. I imported them with Android Studio.

> I will also need you to rebase your branch onto my branch. You have instead merged my branch into yours, which means that if I merge this as-is, the commits I made will be duplicated.

My bad. Do you know if there is any way to fix this? Or I need to revert to a backup copy I made before doing this?

> I think the "transfer" cards are a little too tall. Most of the design is pretty compact and this seems a bit out of place.

I will try even more models, but in my opinion, it is not such a big change to annoy you visually

**Comment by slowscript** [comment]
Did you import them as SVGs? Android Studio acts weirdly with some SVGs. If that's the case, just export them from the software you made them in as a large enough PNG and import that. That's what I did with my icons (even though better solutions likely exist).

If you have a backup right before the merge, it would be probably be best to revert to that and do the rebase. If not, then what I (not a git expert by any means) would do is that 1) I would turn the commits into patches, 2) start with a fresh clone and 3) reapply and commit them one by one. This would also allow you to fix some of the things without making additional commits. Anyway, Google is your friend in this case. I could also try to fix it for you if you want to (but I can't guarantee it will work).

**Comment by rares45** [comment]
> Did you import them as SVGs?
No. I imported them from Android Studio with Android Studio. They are like the official icon package for Android apps. (they are under the Apache License 2.0 so we can use them).

I am gonna fix them my self. Cuz I wanna fix some things.

**Comment by rares45** [comment]
@slowscript I fixed it. I fixed the tint problem, the rebase problem and the typo.

**Comment by slowscript** [comment]
Awesome!

About the other things: I've played around with the icons and it seems that no matter what I try, Android Studio still renders them weirdly. I guess this doesn't really matter. What I did find out however is that the PNGs are redundant. Just import the icons as a Vector Asset (instead of an Image Asset). This way only a single XML and no broken PNGs are generated.

About the design: I still think that the "transfer" cards should be a little less tall - I'd say something around 80dp and corner radius of 10dp would be ideal. This is a non-issue though. I will probably tweak this along with some other things later.

The only thing stopping the merge now are the images. I don't want to have a bunch of useless broken PNGs in the repository.

**Comment by rares45** [comment]
@slowscript I removed the pngs in the force push. And now the app uses the xml.

About the transfer card, I will tweak them. I'm gonna continue working on this next week. I think I forgot about some icons that need a tint and I have some drawing boards with even more modifications (I am not a UI designer, front-end developer or anything like that btw.). And even more translations.

After merging if you want to contact me you can do it at the email address on my profile.

**Comment by slowscript** [comment]
Looks good to me now

**Pull request merged**